compiler/runtime/Runtime.hpp

```diff
@@ -505,7 +505,8 @@ typedef struct TR_AOTMethodHeader {
 #define TR_AOTMethodHeader_NeedsRecursiveMethodTrampolineReservation 0x00000001
 #define TR_AOTMethodHeader_IsNotCapableOfMethodEnterTracing          0x00000002
 #define TR_AOTMethodHeader_IsNotCapableOfMethodExitTracing           0x00000004
-
+#define TR_AOTMethodHeader_UsesEnableStringCompressionFolding        0x00000008
+#define TR_AOTMethodHeader_StringCompressionEnabled                  0x00000010
 
```

Comment:
That sounds like a good plan.  `Runtime.hpp` was one of the few files that received little attention prior to the initial compiler technology code contribution so there likely is some low-hanging refactoring fruit in there.  You can close this pull request.